### PR TITLE
feat: Update did-jwt to 8.x.x, secp256k1 to 5.x.x [DEV-4255]

### PR DESCRIPTION
**Pull request by DaevMithran**

### CODE REVIEW

**Comment by ankurdotb** [comment]
Task linked: [DEV-4255 Backport maintenance release components selectively to v2.x major version](https://app.clickup.com/t/6600954/DEV-4255)

File: package.json [hunk 1]
```diff
@@ -57,30 +57,32 @@
 		"@cosmjs/tendermint-rpc": "~0.30.0",
```
Comment (ankurdotb) [inline package.json:57]: Shouldn't we be bumping CosmJS dependencies as well?

File: package.json [hunk 2]
```diff
@@ -57,30 +57,32 @@
...
+		"@semantic-release/release-notes-generator": "^14.0.1",
+		"@types/jest": "^29.5.12",
+		"@types/node": "^20.16.1",
+		"@types/uuid": "^10.0.0",
+		"conventional-changelog-conventionalcommits": "^8.0.0",
+		"jest": "^29.7.0",
+		"prettier": "^3.3.3",
+		"ts-jest": "^29.1.2",
+		"ts-node": "^10.9.2",
+		"typescript": "^5.5.4",
```
Comment (ankurdotb) [inline package.json:85]: Hmm I think it's missing the plain vanilla semantic-release package actually. The version would be 24.1.0 I believe on latest.

The above versions here for the dev-deps for semantic release and conventional commit are for Node v20.x. We should also look at bumping node version in package.json here and see if that works

**Review by ankurdotb — COMMENTED** [review]
Also consider bumping ts-proto within the 2.x family

File: package.json (hunk 1 above)
Comment (DaevMithran) [reply package.json:57]: No they are not compatible with react native which is a requirement for credo. That's the reason we have release 2.x

File: package.json (hunk 2 above)
Comment (DaevMithran) [reply package.json:85]: Credo supports `node>=18`

**Pull request merged**